### DISCUSSION TITLE
Annoying "/true" from [webpack-dev-middleware] wait until bundle finished

### DISCUSSION THREAD

**Discussion by AndresMpa** [webpack/webpack#17689] [Q&A]
Hey there, I'm getting offset about this Miss configuration? I don't why it this happening, but every time I use `npm run dev` from my package.json I get `http://localhost:8080/true` but `/true` doesn't exists so I get a `Cannot GET /true`. I have look for info about it but I haven't found any solution, do you guys know how can I fix this? I'm getting crazy with it...

My package.json
```json
 "scripts": {
    "dev": "webpack serve --mode development --open true --config ./webpack.config.dev.js"
  },
```

My webpack.config.dev.js
```js
 devServer: {
    watchFiles: ['./src/**/*', './public/**/*'],
    liveReload: true,
    compress: true,
    hot: true,
    port: process.env.PORT,
  },
```

Also this is my entire [webpack.config.dev.js](https://github.com/AndresMpa/babylon/blob/professionalStateManagementWithReact/professionalStateManagementWithReact/webpack.config.dev.js)

**Comment by alexander-akait** [comment]
Sorry for delay, the problem still valid?

**Reply by nyngwang** [reply]
I believe the problem is not related to webpack. I don't have it with the same settings.
